### ISSUE TITLE
Tefnut's Call can store unlimited power

### ISSUE THREAD

**Issue by GoldenTotem** [TeamMetallurgy/Atum2#419]
Minecraft 1.16.5  Atum 2.2.12  Forge 36.2.35

Tefnut's Call can have more damage if you holding right click more longer and the time has no limitation.
For example, holding Tefnut's Call for 34 seconds and throw it to an iron golem, it will be killed instantly.

**Comment by GirafiStudios** [comment]
It's not quite unlimited, but it's indeed insanely high. Fixed in-dev for newer version that (hopefully) will be released eventually.

I can unfortunately not fix it for 1.16.5, as the old Forge repositories is no longer there. There is ways to get it setup with a modern Gradle setup however, but they require you to either use the offical or Parchment mappings. There is no support for the old MCP mappings, which Atum is still using for 1.16.5

**Comment by GoldenTotem** [comment]
I have fix it using crafttweaker script, which make tefnut's call has the same using with trident. Hope that may be helpful.
```zenscript
CTEventManager.register<MCLivingEntityUseItemTickEvent>(event => {
    if (event.item.definition == <item:atum:tefnuts_call>.definition) {
        if (event.duration <= 7191) event.duration = 7200 - 55;
    }
});
```
